ChromecastSampleApp/ChromecastSampleApp/Players/SkinPlayerViewController.m

```diff
@@ -0,0 +1,92 @@
+@property (nonatomic) IBOutlet UIView *videoView;
+@property (nonatomic) OOSkinViewController *skinController;
+@property (nonatomic) OOCastManager *castManager;
+
+@end
+
+@implementation SkinPlayerViewController
+
+#pragma mark - Life cycle
+
+- (void)viewDidLoad {
+  [super viewDidLoad];
+
+  self.title = self.mediaInfo.title;
+  NSLog(@"%@s", self.mediaInfo.domain);
+
+  OOOptions *options = [OOOptions new];
+  OOOoyalaPlayer *ooyalaPlayer = [[OOOoyalaPlayer alloc] initWithPcode:self.mediaInfo.pcode
+                                                                domain:[[OOPlayerDomain alloc] initWithString:self.mediaInfo.domain]
+                                                               options:options];
+  OODiscoveryOptions *discoveryOptions = [[OODiscoveryOptions alloc] initWithType:OODiscoveryTypePopular
+                                                                            limit:10
+                                                                          timeout:60];
+  NSURL *jsCodeLocation = [NSBundle.mainBundle URLForResource:@"main" withExtension:@"jsbundle"];
+  //  NSURL *jsCodeLocation = [NSURL URLWithString:@"http://localhost:8081/index.ios.bundle?platform=ios"];
+  NSDictionary *overrideConfigs = @{@"upNextScreen": @{@"timeToShow": @"8"}};
+
+  ooyalaPlayer.actionAtEnd = OOOoyalaPlayerActionAtEndPause;  //This is recommended to make sure the endscreen shows up as expected
+  OOSkinOptions *skinOptions = [[OOSkinOptions alloc] initWithDiscoveryOptions:discoveryOptions
+                                                                jsCodeLocation:jsCodeLocation
+                                                                configFileName:@"skin"
+                                                               overrideConfigs:overrideConfigs];
+
+  _skinController = [[OOSkinViewController alloc] initWithPlayer:ooyalaPlayer
+                                                     skinOptions:skinOptions
+                                                          parent:self.videoView
+                                                   launchOptions:nil];
+  [self addChildViewController:_skinController];
+  _skinController.view.frame = self.videoView.bounds;
+
+  [NSNotificationCenter.defaultCenter addObserver:self
+                                         selector:@selector(notificationHandler:)
+                                             name:nil
+                                           object:self.skinController];
+
+  [NSNotificationCenter.defaultCenter addObserver:self
+                                         selector:@selector(notificationHandler:)
+                                             name:nil
+                                           object:ooyalaPlayer];
+  [ooyalaPlayer setEmbedCode:self.mediaInfo.embedCode];
+
+  _castManager = [OOCastManagerFetcher fetchCastManager];
+  [ooyalaPlayer initCastManager:self.castManager];
+  self.castManager.notifyDelegate = self.skinController.castNotifyHandler;
```

Comment:
I'm getting build errors here, could you make sure 'notifyDelegate' and 'castNotifyHandler' are visible and reachable?

Reply:
Because you need changes both from `OoyalaSDK` and `OoyalaSkinSDKz